### PR TITLE
Matter Temperature Sensor

### PR DESCRIPTION
**Pull request by lboue**
Add Matter Temperature Sensor:
- Cluster
- Device type

### CODE REVIEW

File: circuitmatter/clusters/general/temperature_measurement.py
Comment (tannewt) [inline circuitmatter/clusters/general/temperature_measurement.py:1]: Please move this to `clusters/measurement`. I want to mirror the chapter structure of the application clusters spec.

File: circuitmatter/clusters/general/temperature_measurement.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from circuitmatter import data_model
+
+
+class TemperatureMeasurement(data_model.Cluster):
+    CLUSTER_ID = 0x0402
```
Comment (tannewt) [inline circuitmatter/clusters/general/temperature_measurement.py:5]:
```suggestion
    CLUSTER_ID = 0x0402
    REVISION = 4
```

File: circuitmatter/device_types/measurement/temperature_sensor.py
Comment (tannewt) [inline circuitmatter/device_types/measurement/temperature_sensor.py:1]: Let's do `device_types/sensor` to match "Chapter 7. Sensor Device Type".

**Review by tannewt — CHANGES_REQUESTED** [review]
Thanks for doing this! Very excited to see it.

I've got a few small suggestions to better match the specs.

File: circuitmatter/clusters/general/temperature_measurement.py
Comment (lboue) [reply circuitmatter/clusters/general/temperature_measurement.py:1]: Good idea

**Comment by lboue** [comment]
Closing this in favor of #21.

**Pull request closed without merging**